de-lib/src/main/java/org/iplantc/de/apps/client/views/tools/ManageToolsViewImpl.java

```diff
@@ -108,7 +119,24 @@ public String getPath() {
         ColumnConfig<Tool, String> tag = new ColumnConfig<Tool, String>(new ValueProvider<Tool, String>() {
             @Override
             public String getValue(Tool object) {
-                return object.getContainer()!=null ? object.getContainer().getImage().getTag() : "";
+                return (object.getContainer()!=null)? object.getContainer().getImage().getTag() : "";
+            }
+
+            @Override
+            public void setValue(Tool object, String value) {
+
+            }
+
+            @Override
+            public String getPath() {
+                return null;
+            }
+        },50, appearance.tag());
+
+        ColumnConfig<Tool, String> status = new ColumnConfig<Tool, String>(new ValueProvider<Tool, String>() {
+            @Override
+            public String getValue(Tool object) {
+                return (object.isPublic() == true)? "Public" : object.getPermission();
```

Comment:
`object.isPublic() == true` is redundant. You could just check `object.isPublic() ?`